scripts/translate_locales.py

```diff
@@ -147,9 +147,9 @@ def process_json_files(directory, target_language, keys_to_update=None):
 
     for filename in os.listdir(directory):
         if (
-            filename.endswith(".json")
-            and filename != "en.json"
-            and filename != "keys_context.json"
+                filename.endswith(".json")
```

Comment:
👀

Reply:
Reverted, my IDE didn't take the setup for python in this project.